Retroactive add signature with video (#29)
CarterKoala/wca-regulations-wrc: wca-guidelines.md

- 11e1+) [REMINDER] The extra attempt must be scrambled using an unmodified scramble sequence generated by an official scramble program (see [Regulation 4f](regulations:regulation:4f), [Guideline 4f+](guidelines:guideline:4f+)).
- 11e2+) [ADDITION] If an extra attempt itself leads to an additional extra attempt, the competitor should keep doing extra attempts until there is a valid extra attempt to replace the original regularly numbered attempt.
- 11e2++) [EXAMPLE] Suppose a competitor has 5 regularly numbered attempts, and #2 leads to extra attempts. The competitor should do extra attempts until there is a valid extra attempt to replace #2. Suppose later another attempt leads to extra attempts, the competitor should proceed in a similar way as for #2.

> - 11f+) [ADDITION] If a missing signature from a new competitor is found and is not possible to be added(e.g. the competitor has left the venue or the compeition has ended), it may be added retroactive with a support of video evidence at the discretion of the WCA Delegate (see [Guideline A7g++++](guidelines:guideline:A7g++++)).

Reviewer: ```suggestion
- 11f+) [ADDITION] If a missing signature from a new competitor is found and cannot otherwise be added (e.g. the competitor has left the venue or the competition has ended), it may be added retroactively with the support of video evidence at the discretion of the WCA Delegate (see [Guideline A7g++++](guidelines:guideline:A7g++++)).
```

Why is A7g a relevant cross-reference? I get that we're trying to apply the same laxity, but this isn't giving an extra attempt, it's effectively lifting a penalty, which A7g++ doesn't allow.

Reviewer: As worded this also only allows the signature to be added, which won't actually be possible if the competitor/judge has gone. 

My alternative solution, rewording A7c5 as follows would address this (and also doesn't restrict to newcomers)

_Missing signatures may be added by the judge and/or the competitor, at the discretion of the WCA Delegate. If the judge and/or competitor are no longer able to physically sign the scorecard, because they have left the competition, or the competition has finished, their attestation that the result was correctly recorded may be accepted as a signature._
